content/en/docs/Reference/catalog-templates.md

```diff
@@ -503,3 +503,357 @@ schema: olm.channel
+| ------------------- | -------------------------------------------------------------------------------------- |
+| -h, --help          | prints help/usage information                                                          |
+| -o, --output <type> | the output format, can be `yaml` or `json`                                             |
+| -c, --composite-config <file> | File to use as the composite configuration file (default "catalog/config.yaml") |
+| --skip-tls-verify   | skip TLS certificate verification for container image registries while pulling bundles |
+| --use-http          | use plain HTTP for container image registries while pulling bundles                    |
+| --validate          | whether or not the created FBC should be validated (i.e 'opm validate') (default true) |
+| --container-tool <tool> | container tool to be used when rendering veneers (should be an equivalent replacement to docker - similar to podman) (default "docker") |
+
+## Specification
+The composite veneer consists of the following schemas:
+- `olm.composite`
+    - This schema defines what, where, and how to build a particular contribution
+- `olm.composite.catalogs`
+    - This schema defines a catalog and the constraints for building the catalog
+
+The `olm.composite.catalogs` cue schema is:
+```cue
+#CatalogConfig: {
+	Schema: "olm.composite.catalogs"
+	Catalogs: [... #Catalog]
+}
+
+#Catalog: {
+	Name:        string
+	Destination: #CatalogDestination
+	Builders: [...string]
+}
+
+#CatalogDestination: {
+	BaseImage:  string
+	WorkingDir: string
+}
+```
+
+The `olm.composite` cue schema is:
+```
+#CompositeConfig: {
+	Schema: "olm.composite"
+	Components: [...#Component]
+}
+
+#Component: {
+	Name:        string
+	Destination: #ComponentDestination
+	Strategy:  #BuildStrategy
+}
+
+#ComponentDestination {
+	Path: string
+}
+
+#BuildStrategy: {
+	Name:   string
+	Template: #TemplateDefinition
+}
+
+#TemplateDefinition: {
+	Schema: string
+	Config: _ @go(,json.RawMessage)
+}
+```
+
+### Examples
+**Catalog Config File**
+
+The following example creates a catalog configuration that:
+- Defines the `v4.10` catalog and restricts contributions to the catalog to only being `basic` templates
+- Defines the `v4.11` catalog and restricts contributions to the catalog to only being `semver` templates
+- Defines the `v4.12` catalog and restricts contributions to the catalog to only being raw FBC
+- Defines the `v4.13` catalog and restricts contributions to the catalog to use a custom workflow
+
+```yaml
+schema: olm.composite.catalogs
+catalogs:
+- name: v4.10 {{< code_callout 1 >}}
+  destination:
+    baseImage: quay.io/operator-framework/opm:v1.24
+    workingDir: catalogs/v4.10 {{< code_callout 2 >}}
+  builders:
+    - olm.builder.basic {{< code_callout 3 >}}
+- name: v4.11
+  destination:
+    baseImage: quay.io/operator-framework/opm:v1.25
+    workingDir: catalogs/v4.11
+  builders:
+    - olm.builder.semver {{< code_callout 4 >}}
```

Comment:
would be good to show that this can be a list of builders.
```suggestion
    - olm.builder.basic
    - olm.builder.semver {{< code_callout 4 >}}
```